crm_compassion/wizards/partner_log_interaction_wizard.py

```diff
@@ -52,3 +53,14 @@ def log_interaction(self):
                 .id,
             }
         )
+        # Create the mail.tracking.email object in the case of a manually logged email
+        email = mail._send_prepare_values(partner=self.partner_id)
+        vals = mail._tracking_email_prepare(self.partner_id, email)
+        vals.update({
+            "state": "sent"
+        })
+        mail_tracking_obj = self.env['mail.tracking.email']
+        mail_tracking_obj.search([
+            ('mail_id', '=', mail.id)
+        ]).unlink()
+        mail_tracking_obj.sudo().create(vals)
```

Comment:
Shouldn't you return the mail?

Reply:
It's not the mail that is created at the end, but a mail_tracking_email and it's automatically linked to the mail created at the beginning. And as it's "fake emails", no need to send or return them, creating them is enough